azure-kusto-data/azure/kusto/data/request.py

```diff
@@ -324,14 +329,81 @@ def _execute(self, endpoint, database, query, default_timeout, properties=None):
 
         raise KustoServiceError([response.json()], response)
 
+    def execute_streaming_ingest(
+        self,
+        database,
+        table,
+        stream,
+        stream_format,
+        mapping_name=None,
+        accept=None,
+        accept_encoding=None,
+        connection=None,
+        content_length=None,
+        content_encoding=None,
+        expect=None,
+    ):
+        """Executes streaming ingest against this client.
+        :param str database: Target database.
+        :param str table: Target table.
+        :param stream: File-like object contains the data to ingest.
+        :param DataFormat stream_format: Format of the data in the stream.
+        :param str mapping_name: Pre-defined mapping of the table. Required when stream_format is json/avro.
+        Other optional params: optional request headers as documented at:
+            https://kusto.azurewebsites.net/docs/api/rest/streaming-ingest.html
+        """
+
+        assert isinstance(database, str), "Expected str instance, found {}".format(type(database))

```

Comment:
we usually don't assert types, just assume it is a string (or at least behaves like one) as it doesn't really matter what type it is as long as it behaves like a string.. I would remove these..